btk/measure.py

```diff
@@ -37,6 +41,74 @@ def get_deblended_images(self, data, index):
         return None
 
 
+class BasicMeasureParams(Measurement_params):
```

Comment:
```suggestion
class BasicMeasureParams(MeasurementParams):
```
+ need to change the main class definition